docs/pages/enroll-resources/database-access/auto-user-provisioning/mysql.mdx

```diff
@@ -3,6 +3,26 @@ title: MySQL Automatic User Provisioning
 description: Configure automatic user provisioning for MySQL.
 ---
 
+Teleport can automatically create an account in your MySQL database when a
+Teleport user connects to the database, removing the need for creating
+individual user accounts in advance or using the same set of shared database
+accounts for all users. This guide shows you how to set up automatic user
+provisioning for MySQL.
+
+## How it works
+
+Before enrolling a MySQL database with Teleport, you set up an admin user on the
+database with permissions to manage users. You then enroll the database in your
+Teleport cluster with a configuration field that includes the name of the admin
+user. 
+
+When a Teleport user connects to the database with a role that enables automatic
+user provisioning, the Teleport Database Service first connects to the database
+as the admin user, creates a temporary user in the database, then initiates a
+database session for the Teleport user as the temporary database user. When the
+Teleport user closes the session, the Teleport Database Service removes the
+temporary database user.
```

Comment:
Overall looks good. 

> When the Teleport user closes the session, the Teleport Database Service removes the temporary database user.

This is true in the `best_effort_drop` mode where the temp user is dropped. In the `keep` mode, for comparison, the temp user is disabled and its privileges are stripped.